### ISSUE TITLE
[Bug]: Server Error 500 - on nonexistent channels in download queue

### ISSUE THREAD

**Issue by alteriks** [tubearchivist/tubearchivist#431]
### I've read the documentation

- [X] I'm running the latest version of Tube Archivist and have read the [release notes](https://github.com/tubearchivist/tubearchivist/releases/latest).
- [X] I have read through the [wiki](https://github.com/tubearchivist/tubearchivist/wiki) and the [readme](https://github.com/tubearchivist/tubearchivist#installing-and-updating), particularly the [common errors](https://github.com/tubearchivist/tubearchivist#common-errors) section.

### Operating System

Linux/Docker

## Describe the bug
TA will give 'Server Error 500' when using channel filter in download queue which hasn't been created yet.

## Steps To Reproduce
1) Using new TA instance add at least one video from channel which wasn't created earlier.
![image](https://user-images.githubusercontent.com/6512767/219076420-a7b5f38f-c635-408b-9f80-90e7edeab124.png)
2) Add at least two videos from different channels and change channel filter from 'all' to one of the newly added, yet nonexistent channel.
![image](https://user-images.githubusercontent.com/6512767/219077000-616ce0f4-7b98-471c-b09e-675665eaa3a6.png)

3) Server Error (500) 
https://tube.alt.lan/downloads/?channel=UCSMOQeBJ2RAnuFungnQOxLg
![image](https://user-images.githubusercontent.com/6512767/219077090-c8f3a7a0-f4bf-43bd-a050-9e2e41e49cdd.png)

https://tube.alt.lan/downloads/?channel=UC7aW7chIafJG6ECYAd3N5uQ
![image](https://user-images.githubusercontent.com/6512767/219077174-044502c9-d743-488c-9276-9e041fba0649.png)

4) Download one of the videos in queue. Channels gets created during process and filter works as expected
![image](https://user-images.githubusercontent.com/6512767/219081768-f168985f-b890-44ee-8804-c7805e5affef.png)

## Expected behavior
Download queue needs to filter properly videos in queue even if channel doesn't exist yet.

### Relevant log output

```shell
tubearchivist    | {"_index":"ta_channel","_id":"UC7aW7chIafJG6ECYAd3N5uQ","found":false}
tubearchivist    | [pid: 35|app: 0|req: 246/246] 192.168.1.5 () {66 vars in 1302 bytes} [Wed Feb 15 17:03:11 2023] GET /downloads/?channel=UC7aW7chIafJG6ECYAd3N5uQ => generated 145 bytes in 36 msecs (HTTP/1.0 500) 7 headers in 248 bytes (1 switches on core 0)
```

**Comment by bbilly1** [comment]
Very interesting, thanks for debugging and collecting your findings here. I can reproduce this.

**Comment by bbilly1** [comment]
Thanks for reporting. This is now fixed in v0.3.3.
